chapter1/chapter1.tex

```diff
@@ -129,17 +132,17 @@ \subsection{Sample spaces and events}
 %$ A^{C_{\Omega}} \in \mathcal{A} $ by item 3. Item 4 then implies that $ \Omega \in \mathcal{A} $.
 \end{Exercise}
 
-The fact that our event spaces are closed under the set complement operation is very convenient. Say I
+The fact that event spaces are closed under the set complement operation is very convenient. Say I
 organized a dinner party and invited $ 10 $ people. The day after you ask me if more than $ 8 $ people
 actually showed up. I just answer that I was very disappointed that my friends Mary and Paul did 
 not come. Although I did not directly address your question you know that the answer is negative. After
 all, I informed you that the complement event of the event you asked about had occurred.
 
 \begin{Exercise} 
-In the above party example, what is the sample space? What is the smallest possible event space?
-%Solution: $ \Omega = \{x_{1} \ldots x_{10} | x_{i} \in \{0,1\}\} $
-%$ \mathcal{A} = \{\Omega, \emptyset, \{\omega \in \Omega | \sum x_{i} > 8\},
-%\{\omega \in \Omega | \sum x_{i} \leq 8\}\} $ 
+In the above party example, what is the sample space? What is the smallest possible event space (in order to...)?
```

Comment:
I think the exercise is not totally unambiguous, we should be  a little more specific here